routes/api/v1/favorites.js

```diff
@@ -41,4 +42,17 @@ router.post('/', (request, response) => {
       });
 });
 
+router.get('/', (request, response) => {
+  database('favorites').select()
```

Comment:
@zacisaacson  did you try selecting the columns in the Knex query? That worked for a single resource and helped me avoid creating a helper method
```suggestion
  database('favorites').select('id', 'title', 'artistName', 'genre', 'rating')
```